packages/graphback-codegen-schema/src/definitions/schemaDefinitions.ts

```diff
@@ -1,45 +1,55 @@
-import { GraphQLInputObjectType, GraphQLFloat, GraphQLList, GraphQLBoolean, GraphQLInt, GraphQLString, GraphQLID, GraphQLEnumType, GraphQLObjectType, GraphQLNonNull, GraphQLField, getNamedType, isScalarType, GraphQLInputFieldMap, GraphQLScalarType, GraphQLNamedType, GraphQLInputField, isSpecifiedScalarType, isEnumType } from "graphql";
+import { GraphQLInputObjectType, GraphQLList, GraphQLBoolean, GraphQLInt, GraphQLString, GraphQLID, GraphQLEnumType, GraphQLObjectType, GraphQLNonNull, GraphQLField, getNamedType, isScalarType, GraphQLInputFieldMap, GraphQLScalarType, GraphQLNamedType, GraphQLInputField, isSpecifiedScalarType, isEnumType } from "graphql";
 import { GraphbackOperationType, getInputTypeName, getInputFieldName, getInputFieldType, isOneToManyField, getPrimaryKey } from '@graphback/core';
 
 const PageRequestTypeName = 'PageRequest';
 const SortDirectionEnumName = 'SortDirectionEnum';
 const OrderByInputTypeName = 'OrderByInput';
 
 const getScalarInputName = (type: GraphQLNamedType) => {
-  if (isSpecifiedScalarType(type)) {
-    return `${type.name}Input`;
+  if (isEnumType(type)) {
+    return `StringInput`
   }
+  
+  return `${type.name}Input`
+}
+
+
+
+export const createInputTypeForCustomScalar = (scalarType: GraphQLScalarType) => {
+  const newInput = new GraphQLInputObjectType({
+    name: getScalarInputName(scalarType),
+    fields: {
+      ne: { type: scalarType },
+      eq: { type: scalarType },
+      le: { type: scalarType },
+      lt: { type: scalarType },
+      ge: { type: scalarType },
+      gt: { type: scalarType },
+      in: { type: GraphQLList(scalarType) },
+      between: { type: GraphQLList(scalarType) }
+    }
+  })
 
-  return 'StringInput';
+  return newInput;
 }
 
-export const FloatScalarInputType = new GraphQLInputObjectType({
-  name: getScalarInputName(GraphQLFloat),
-  fields: {
-    ne: { type: GraphQLFloat },
-    eq: { type: GraphQLFloat },
-    le: { type: GraphQLFloat },
-    lt: { type: GraphQLFloat },
-    ge: { type: GraphQLFloat },
-    gt: { type: GraphQLFloat },
-    in: { type: GraphQLList(GraphQLFloat) },
-    between: { type: GraphQLList(GraphQLFloat) }
-  }
-})
+export const createInputTypeForEnum = (scalarType: GraphQLScalarType) => {
+  const newInput = new GraphQLInputObjectType({
```

Comment:
The input object fields looks to be the same as that returned in `createInputTypeForCustomScalar`

Reply:
I am not sure if this is even used?  I think we should treat enumInputs like a StringInput, WDYT? @wtrocki

Reply:
Yep. I pushed that by mistake. We should use strings

Reply:
Okay, will remove this.